db.js

```diff
@@ -14,9 +14,10 @@ const testDB = async () => {
 const devDB = async () => {
   const db = new Low(new JSONFile('localdb.json'))
   await db.read()
-  // Migrate devDB data by updating seedData.js
-  db.data = seedData
-  await db.write()
+  if (!db.data) {
+    db.data = seedData
+    await db.write()
+  }
```

Comment:
Does this revert the migration bit which we added [here](https://github.com/checkr/oauth-reference-integration/pull/32)? I'd be ok with that since we don't expecting to be changing things anytime soon.

Reply:
I moved the migration function to its own function that is run before the express server is started. See [here](https://github.com/checkr/oauth-reference-integration/pull/41/files#diff-a4c65ede64197e1a112899a68bf994485b889c4b143198bac4af53425b38406fR42)